app/qml/settings/MMSettingsPage.qml

```diff
@@ -109,20 +118,10 @@ MMPage {
         value: __appSettings.intervalType === MM.StreamingIntervalType.Distance ? qsTr("Distance Traveled") : qsTr("Time elapsed")
         selected: [__appSettings.intervalType]
 
-        model: ListModel {
-          ListElement {
-            value: MM.StreamingIntervalType.Time
-            text: qsTr("Time elapsed")
-          }
-          ListElement {
-            value: MM.StreamingIntervalType.Distance
-            text: qsTr("Distance traveled")
-          }
-        }
+        model: root.createIntervalTypeModel()
```

Comment:
Can you make description here saying that to use C++ enums you need to create model dynamically?